src/rgw/rgw_sal_motr.cc

```diff
@@ -3359,7 +3359,9 @@ std::string MotrStore::zone_unique_id(uint64_t unique_num)
 
 std::string MotrStore::zone_unique_trans_id(const uint64_t unique_num)
 {
-  return "";
+  static uint64_t num = 0xffffffff;
+  num = num -1;
```

Comment:
If multiple threads call it concurrently - the result may end up with the same `num`.